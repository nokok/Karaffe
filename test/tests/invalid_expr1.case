tests/test_resources/neg/InvalidExpr1.krf
