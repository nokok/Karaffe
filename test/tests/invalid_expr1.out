#REGEX Syntax Error at 2:0 .*InvalidExpr1.krf$
missing IntegerLiteral at '<EOF>'
